src/utils/custom-hooks.js

```diff
@@ -147,66 +148,97 @@ export const useMultipleDeferredFetch = (
     hasResult = true
 ) => {
     const initialState = {
-        status: FetchStatus.IDLE,
-        errorMessage: [],
-        paramsOnError: [],
-        data: [],
+        public: {
+            status: FetchStatus.IDLE,
+            errorMessage: [],
+            paramsOnError: [],
+            data: [],
+        },
+        counter: 0,
     };
 
     const [state, dispatch] = useReducer((lastState, action) => {
         switch (action.type) {
             case FetchStatus.IDLE:
-                return { ...initialState };
+                return {
+                    ...initialState,
+                    counter: 0,
```

Comment:
`initialState` already has counter to 0 no ?

Reply:
fixed